packages/sqlgg/sqlgg.20231201/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "ygrek@autistici.org"
+authors: ["ygrek"]
+homepage: "https://ygrek.org/p/sqlgg/"
+dev-repo: "git+https://github.com/ygrek/sqlgg.git"
+bug-reports: "https://github.com/ygrek/sqlgg/issues"
+license: "GPL-2.0-only"
+tags: [ "org:ygrek" ]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs "@install" "@doc" {with-doc}]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.0"}
+  "menhir"
+  "mybuild"
```

Comment:
It seems that mybuild does not like the following distros and are failing the CI jobs.
But those are not directly related to this PR and I think we can safely ignore them.

- arm32-ocaml-5.1
- ocaml-5.1-afl
- ppc64-ocaml-5.1
- x86_32-ocaml-5.1